### PR TITLE
Don't minify __stack_pointer import

### PR DESCRIPTION
**Pull request by sbc100**
This is core import like __memory_base and __table_base.

### CODE REVIEW

File: src/passes/MinifyImportsAndExports.cpp [hunk 1]
```diff
@@ -145,7 +145,7 @@ struct MinifyImportsAndExports : public Pass {
     std::map<Name, Name> oldToNew;
     auto process = [&](Name& name) {
       // do not minifiy special imports, they must always exist
-      if (name == MEMORY_BASE || name == TABLE_BASE) {
+      if (name == MEMORY_BASE || name == TABLE_BASE || name == STACK_POINTER) {
```
Comment (kripken) [inline src/passes/MinifyImportsAndExports.cpp]: are these "ABI imports" documented in tool-conventions? if so a link to there might be nice.

for the first two I think the dynamic linking doc there mentions them.

Comment (sbc100) [reply src/passes/MinifyImportsAndExports.cpp]: I don't know .. all i know is that if I allow minification of __stack_pointer I get a test failure.  To be honest I'm not sure why these can't be minified too.

Comment (sbc100) [reply src/passes/MinifyImportsAndExports.cpp]: Oh you are correct this should be documented in https://github.com/WebAssembly/tool-conventions/blob/master/DynamicLinking.md but isn't yet.

**Comment by sbc100** [comment]
Any idea why binaryen travis checks take so long?  It seems like we should be able to get good test coverage in less than an hour.

**Comment by kripken** [comment]
Binaryen's travis is shared with all other `WebAssembly/` jobs, and the free tier is just not great when there is a lot of activity I think...

**Pull request merged**